app/services/invoices/payments/adyen_service.rb

```diff
@@ -98,12 +98,8 @@ def update_payment_method_id
           Lago::Adyen::Params.new(payment_method_params).to_h,
         ).response
 
-        if (payment_method_id = result['storedPaymentMethods']&.first&.dig('id'))
-          customer.adyen_customer.update!(payment_method_id:)
-        end
-      rescue Adyen::AdyenError => e
-        deliver_error_webhook(e)
```

Comment:
we do not send the webhook anymore?

Reply:
We do it here instead: https://github.com/getlago/lago-api/pull/1485/files/85e33c7334cf0f76a8b4355ea65006a059a0de39#diff-43fd964e38ba8c7b757a07adaae23508fbece49763232b081bc36136adb041deR110